tendrl/commons/objects/cluster/atoms/delete_cluster_details/__init__.py

```diff
@@ -0,0 +1,113 @@
+import etcd
+import json
+
+from tendrl.commons import objects
+from tendrl.commons.utils import log_utils as logger
+
+
+class DeleteClusterDetails(objects.BaseAtom):
+    def __init__(self, *args, **kwargs):
+        super(DeleteClusterDetails, self).__init__(*args, **kwargs)
+
+    def run(self):
+        integration_id = self.parameters['TendrlContext.integration_id']
+        cluster_tendrl_context = NS.tendrl.objects.ClusterTendrlContext(
+            integration_id=integration_id
+        ).load()
+
+        etcd_keys_to_delete = []
+        etcd_keys_to_delete.append(
+            "/clusters/%s/Peers" % integration_id
+        )
+        etcd_keys_to_delete.append(
+            "/clusters/%s/Bricks" % integration_id
+        )
+        etcd_keys_to_delete.append(
+            "/clusters/%s/Volumes" % integration_id
+        )
+        etcd_keys_to_delete.append(
+            "/clusters/%s/GlobalDetails" % integration_id
+        )
+        etcd_keys_to_delete.append(
+            "/clusters/%s/TendrlContext" % integration_id
+        )
+        etcd_keys_to_delete.append(
+            "/clusters/%s/Utilization" % integration_id
+        )
+        etcd_keys_to_delete.append(
+            "/clusters/%s/raw_map" % integration_id
+        )
+        etcd_keys_to_delete.append(
+            "/alerting/clusters/%s" % integration_id
+        )
+        etcd_keys_to_delete.append(
+            "/indexes/tags/detected_cluster/%s" %
+            cluster_tendrl_context.cluster_id
+        )
+        etcd_keys_to_delete.append(
+            "/indexes/tags/detected_cluster_id_to_integration_id/%s" %
+            cluster_tendrl_context.cluster_id
+        )
+        etcd_keys_to_delete.append(
+            "/indexes/tags/provisioner/%s" % integration_id
+        )
+        etcd_keys_to_delete.append(
+            "/indexes/tags/tendrl/integration/%s" %
```

Comment:
lets clean this tag in integration service

Reply:
done